src/grecon/grecon.c

```diff
@@ -103,7 +103,7 @@ void grecon(struct grecon_conf* param,  const long dims1[DIMS], complex float* o
 		sens1 = cov1;
 	}
 
-	fftmod(DIMS, dims1, FFT_FLAGS, sens1, sens1);
```

Comment:
Ok.The case with caltwo will probably not work with more dimensions, but this will get rewritten later anyway...